apps/src/code-studio/teacherPanelHelpers.js

```diff
@@ -67,13 +64,6 @@ export function queryLockStatus(store, scriptId) {
     $.ajax('/api/lock_status', {
       data: {script_id: scriptId}
     }).done(data => {
-      // Extract the state that teacherSectionsRedux cares about
-      const teacherSections = Object.values(data).map(section => ({
-        id: section.section_id,
-        name: section.section_name
-      }));
-
-      store.dispatch(setSections(teacherSections));
```

Comment:
re: https://codedotorg.slack.com/archives/CA3KCSGTD/p1572553393017000 I was getting the `SET_SECTIONS called multiple times in a way that would remove data` error because I added a call to setSections in progress.js so that I could have the section assignment info I needed for the new dropdown, but `queryLockStatus` was also being called in progress.js which called `setSections` with limited data about the section returned from the lock status api.  So, I removed the call to `setSections` here since it was redundant. Well, it's definitely redundant in my case on the unit overview page, but maybe not everywhere.  If the stage_lock UI test and the stage_lock integration test pass is there anywhere else I should check that might not be tested?

Reply:
I see setSections being dispatched in the following files:
```
code-studio/teacherPanelHelpers.js
sites/studio/pages/teacher_dashboard/show.js
sites/studio/pages/courses/show.js
templates/teacherDashboard/teacherSectionsRedux.js
```
which also sneakily includes `TeacherHomepage` --> `TeacherSections` --> `asyncLoadSectionData` --> `setSections`

I think you've probably covered courses/show and that stage lock tests have covered teacher panel. but I would suggest taking a quick peek at teacher homepage and teacher dashboard to make sure those are not affected, or convincing yourself that existing test coverage is adequate.